app/views/admin/shared/_logo_upload.html.erb

```diff
@@ -1,11 +1,11 @@
  <div class="crayons-field mt-6 align-left" data-controller="logo-upload">
   <%= label_tag :logo, class: "crayons-field__label" do %>
     Logo
-    <span class="crayons-field__required crayons-tooltip" data-tooltip="This will set the logo for your Forem" aria-describedby="logo-subtitle"></span>
+    <span class="crayons-field__required crayons-hover-tooltip" data-tooltip="This will set the logo for your Forem"></span>
```

Comment:
I still can't trigger this with the mouse - it has height/width of 0 🙈

Reply:
@aitchiss, I ended up removing the tooltip like your original suggestion and updated the copy instead. @JennieOcken, for context, this tooltip was not showing up even before the refactor in this PR. If you're good with this wording Jennie, I'll go ahead and merge this my AM.

![CleanShot 2021-12-08 at 16 01 33](https://user-images.githubusercontent.com/833231/145283798-290fddb4-fb46-42c3-a1aa-f6d8e7419f12.png)

Reply:
I love it 🙌 

My anti-tooltip crusade continues 😆

Reply:
Good with no tooltip.

Can we change it to: "Your logo will display in the upper left hand corner of your Forem.  Upload a PNG or JPEG file."